tests/neteventebpfext/neteventebpfext_unit/netevent_ebpfext_unit.cpp

```diff
@@ -25,6 +25,7 @@ CATCH_REGISTER_LISTENER(cxplat_passed_test_log)
 #define MAX_EVENTS_COUNT 1000
 #define NETEVENT_EVENT_TEST_TIMEOUT_SEC 90
 #define NETEVENT_EVENT_STRESS_TEST_TIMEOUT_SEC 90
+#define MAX_PACKET_SIZE 1600
```

Comment:
For my understanding, why 1600 bytes for data_out?

Reply:
Just picked an arbitrary large number for the test